src/vtl.ts

```diff
@@ -418,4 +437,78 @@ export class VTL {
     const __exhaustive: never = node;
     return __exhaustive;
   }
+
+  private mapForEachArgs(call: CallExpr) {
```

Comment:
Should this be called `getForEachArgNames`? Does it need to be a function on the class? Could it be static function?
```ts
/**
 * Returns the [value, index, array] arguments if this CallExpr is a `forEach`, `map` or `reduce` call.
 */
export function getForEachArgNames(call: CallExpr): string[] {

}
```

Reply:
Yeah, it could be static, will rename too.